service/role.go

```diff
@@ -339,6 +364,19 @@ func (r *roleService) updateRoleToken(ctx context.Context, domain, role, proxyFo
 	return rt.(*RoleToken), err
 }
 
+// TODO:
+func roleCacheMemoryUsage(data *cacheData) int64 {
+	tokenSize := int64(unsafe.Sizeof(data.token)) + int64(len(data.token.Token))
+	expiryTimeSize := int64(unsafe.Sizeof(data.token.ExpiryTime)) + int64(len(strconv.FormatInt(data.token.ExpiryTime, 10)))
+	domainSize := int64(unsafe.Sizeof(data.domain)) + int64(len(data.domain))
+	roleSize := int64(unsafe.Sizeof(data.role)) + int64(len(data.role))
+	proxyForPrincipalSize := int64(unsafe.Sizeof(data.proxyForPrincipal)) + int64(len(data.proxyForPrincipal))
+	minExpirySize := int64(unsafe.Sizeof(data.minExpiry))
+	maxExpirySize := int64(unsafe.Sizeof(data.maxExpiry))
+
+	return tokenSize + expiryTimeSize + domainSize + roleSize + proxyForPrincipalSize + minExpirySize + maxExpirySize
```

Comment:
Maybe we can do as the following. Can you help to check they are equal or not?
```
	structSize := int64(unsafe.Sizeof(*data))
	stringSize := int64(len(data.domain) + len(data.role) + ...)
	rtStructSize := int64(unsafe.Sizeof(data.token))
	rtStringSize := int64(len(data.token.Token))
	
	return structSize + stringSize + rtStructSize + rtStringSize
```

Reply:
@WindzCUHK 
fixed in https://github.com/AthenZ/athenz-client-sidecar/pull/18/commits/8363817895e6a2447ba029bf48de8dcf8e404d52